### PR TITLE
added facebook messenger chat files parser

### LINKED ISSUE #14: Implement parser for Facebook Messenger chatfiles
Implement a new parser enabling the processing of Facebook Messenger export files. 
The logic should be aligned with the existing parser classes.

Please find more information about exporting Facebook Messenger chatlogs [here](https://www.facebook.com/help/messenger-app/713635396288741?cms_id=713635396288741&published_only=true).

### PR DESCRIPTION
**Pull request by gajrajgchouhan**
- I couldnt find a way to check the time zone for timestamps.
- Also, the parser is only for JSON.

Fixes #14

### CODE REVIEW

File: chatminer/chatparsers.py [hunk 1]
```diff
@@ -195,6 +197,54 @@ def _get_message_author(self, message):
...
+            'datetime': datetime,
+            'author': mess["sender_name"],
+            'message': ''
+        }
+        if mess['type'] == 'Share':
+           parsed_message['message'] = mess['share']['link']
+        elif 'sticker' in mess:
+            parsed_message['message'] = mess['sticker']['uri']
+        else:
+             parsed_message['message'] = mess['content']
```
Comment (joweich) [inline chatminer/chatparsers.py:240]: W0311: Bad indentation. Found 13 spaces, expected 12 (bad-indentation)

File: chatminer/chatparsers.py [hunk 2]
```diff
@@ -195,6 +197,54 @@ def _get_message_author(self, message):
...
+
+    def _parse_message(self, mess):
+        datetime = datetimemodule.datetime.fromtimestamp(mess['timestamp_ms']/1000)
+        parsed_message = {
+            'datetime': datetime,
+            'author': mess["sender_name"],
+            'message': ''
+        }
+        if mess['type'] == 'Share':
+           parsed_message['message'] = mess['share']['link']
```
Comment (joweich) [inline chatminer/chatparsers.py:236]: W0311: Bad indentation. Found 11 spaces, expected 12 (bad-indentation)

**Review by joweich — CHANGES_REQUESTED** [review]

**Comment by joweich** [comment]
@gajrajgchouhan did you use the guide linked in #14 for exporting the chat?

**Comment by gajrajgchouhan** [comment]
I'll do the requested changes in a bit.

Yes, I exported the chats from the Android messenger app using the specified guide.

**Comment by joweich** [comment]
@gajrajgchouhan feel free to have a look #18 in case you want to support further by adding a unit test for Facebook chatfiles

**Pull request merged**